icons/indeed.svg

```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Indeed icon</title><path d="M14.822 10.308a3.018 3.018 0 111.32-4.038 3.006 3.006 0 01-1.32 4.044zM11.6.572c2.47-.901 5.294-.852 7.408.982a3.587 3.587 0 011.023 1.37c.213.69-.749-.07-.88-.168a9.407 9.407 0 00-2.15-1.095C12.837.386 8.897 2.707 6.463 6.316a19.505 19.505 0 00-2.248 5.126 2.918 2.918 0 01-.213.642c-.107.204-.049-.547-.049-.572a15.821 15.821 0 01.43-2.239C5.511 5.34 8.01 2.065 11.6.565zm.037 20.993v-8.763c.249.025.486.037.736.037a6.167 6.167 0 003.219-.895v9.62c0 .822-.15 1.43-.52 1.826A1.874 1.874 0 0113.62 24a1.825 1.825 0 01-1.427-.609c-.368-.404-.56-1.013-.56-1.825z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Indeed icon</title><path d="M14.8224 10.3074a3.0179 3.0179 0 111.32-4.0379 3.0059 3.0059 0 01-1.32 4.0439zM11.6005.5718c2.47-.901 5.2938-.852 7.4077.982a3.5869 3.5869 0 011.023 1.3699c.213.69-.749-.07-.88-.168a9.4066 9.4066 0 00-2.1499-1.095C12.8375.3857 8.8976 2.7067 6.4637 6.3155a19.5042 19.5042 0 00-2.2479 5.1258 2.9179 2.9179 0 01-.213.642c-.107.204-.049-.547-.049-.572a15.8203 15.8203 0 01.43-2.2389c1.128-3.9328 3.6269-7.2077 7.2167-8.7076zm.037 20.992v-8.7625c.249.025.486.037.736.037a6.1667 6.1667 0 003.2189-.895v9.6196c0 .822-.15 1.43-.52 1.826a1.874 1.874 0 01-1.452.6099 1.825 1.825 0 01-1.4269-.609c-.368-.404-.56-1.013-.56-1.825z"/></svg>
```

Comment:
Looks Great! 💯
Any chance you can reduce it to 3 decimals?

Reply:
I've tried, but no. Even rebuiling from source didn't work. So I guess we have to stick with 4 here.